internal/npm_install/generate_build_file.js

```diff
@@ -15,46 +15,30 @@
  * limitations under the License.
  */
 /**
- * @fileoverview This script generates a BUILD.bazel file by analyzing
+ * @fileoverview This script generates BUILD.bazel files by analyzing
  * the node_modules folder layed out by yarn or npm. It generates
  * fine grained Bazel filegroup targets for each root npm package
  * and all files for that package and its transitive deps are included
- * in the filegroup. For example, `@<workspace>//:jasmine` would
+ * in the filegroup. For example, `@<workspace>//jasmine` would
  * include all files in the jasmine npm package and all of its
  * transitive dependencies.
  *
  * nodejs_binary targets are also generated for all `bin` scripts
- * in each package. For example, the `@<workspace>//:jasmine/jasmine`
+ * in each package. For example, the `@<workspace>//jasmine/bin:jasmine`
  * target will be generated for the `jasmine` binary in the `jasmine`
  * npm package:
  *
  * ```
  * nodejs_binary(
- *   name = "jasmine/jasmine",
- *   entry_point = "jasmine/bin/jasmine.js",
- *   data = [":jasmine"],
+ *     name = "jasmine",
+ *     entry_point = "jasmine/bin/jasmine.js",
+ *     data = ["//jasmine:jasmine"],
```

Comment:
should it be just `//jasmine` ?

Reply:
I'll just remove the code snipped. Its not accurate anyway since the `//jasmine/bin:jasmine` target is an alias and not a `nodejs_binary`